aviary/subsystems/atmosphere/atmosphere.py

```diff
@@ -0,0 +1,71 @@
+import openmdao.api as om
+
+from dymos.models.atmosphere.atmos_1976 import USatm1976Comp
+
+from aviary.subsystems.atmosphere.flight_conditions import FlightConditions
+from aviary.variable_info.enums import SpeedType
+from aviary.variable_info.variables import Dynamic
+
+
+class Atmosphere(om.Group):
+    """
+    Group that contains atmospheric conditions for the aircraft's current flight
+    condition, as well as conversions for different speed types (TAS, EAS, Mach)
+    """
+
+    def initialize(self):
+        self.options.declare(
+            'num_nodes', types=int, desc='Number of nodes to be evaluated in the RHS'
+        )
+
+        self.options.declare(
+            'h_def',
+            values=('geopotential', 'geodetic'),
+            default='geopotential',
+            desc='The definition of altitude provided as input to the component. If '
```

Comment:
I think this should probably be geodetic by default, if that is generally how we report our altitudes.   However, it looks like we have been using dymos' default of "geopotential" throughout aviary.